operating_procedures.md

```diff
@@ -154,283 +111,73 @@ FOCUS Operating Procedures
-
-# 6\. Approval Process
-
-Specification(s) created by FOCUS Groups within the FOCUS Project will approve or reject contributions following a democratic process; the majority. 
-
-This differs from an Open Source organization that normally follows a meritocratic process where the Maintainer decides what to accept or reject and If a person doesn’t like the decision that their contribution is rejected, then they can “fork” the project.
-
-The goal for a specification creation in the FOCUS project is to reach interoperability, therefore “forking” is not the solution to a technical dispute. If there is a sustainable objection in a contribution the resolution is via a vote, see Seeking Consensus.
-
-The Review & Approval process implies that all the contributions need to be accepted by the Working Group.
-
-FOCUS Groups generating Non-Specification outputs will follow the same Review & Approval process, but not specifically avoid members “forking” the project.
-
-## 6.1 Review & Approval Process
-
-### 6.1.1 Review period
-
-Period of time during which the contribution will be under review before being merged.
-The period can be: 0 or some number of days specified by the FG Chair.
-
-0 days imply that the contribution is merged without Working Group review
-Comments or Objections, 0 day review merges MUST only be used for non-material changes to the FOCUS group work product(s).
-
-During the Review & Approval process members MAY raise comments or objections.
-
-Comments MUST be taken in consideration by the Working Group, but they MAY be dismissed if they group thinks that are not relevant.
-
-Objections MUST be taken in consideration and they SHALL NOT be dismissed by the Working Group without being reviewed.
-
-If a contribution receives an objection the group MUST resolve the issue, with the person that raised the objection, before deciding the status of the contribution. If the objection is sustained, meaning the person doesn’t remove it, then the group will have to recur to a vote to resolve it.
-
-### 6.1.2 Approval Criteria
-
-A contribution is considered approved and therefore it can be merged if:
-The contribution has not received any sustainable objection during the review period, AND At least 3 reviewers have indicated that they agree with the contribution
-
-If a sustained objection is received, the contribution cannot be merged, even if 3 or more contributors agreed with the contribution.
-
-If during the review period a contribution receives a comment, it is up to the group or maintainer(s) to accept the comment or not. In any case, in order to merge the contribution at least 3 reviewers MUST indicate that they agree with the contribution.
-
-# 7\. FOCUS Process Flows
+# 4\. FOCUS Working Group (FG) 
 
-## 7.1 Work Packages
+Each FOCUS Working group may have an operational_proceedures.md document in the FOCUS Group GitHub respository that outlines their processes and proceedures.
```

Comment:
related to comment on line 33. I think you want to say that the WGs will follow this OP, and document any special or necessary changes to reflect the work product